src/EditorFeatures/Test/Utilities/PatternMatcherTests.cs

```diff
@@ -383,17 +383,27 @@ public void MatchMultiWordPattern_AsteriskSplit()
         }
 
         [Fact, WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/544628")]
-        public void MatchMultiWordPattern_LowercaseSubstring1()
-            => Assert.Null(TryMatchMultiWordPattern("Operator", "a"));
```

Comment:
where did this test go?  i don't see it anymore.

Reply:
It was testing the behavior that is now changed so I deleted it and added another one that in my opinion express the intent better.

Reply:
please don't delete tests.  update the validation to reflect the new, now expected, behavior.  that allows the test change to demontrate the behavior, instead of having to guess if it did change or not.  Thanks! :)

Reply:
Alright, should I then delete the WorkItem link at least? I obviously can't open it, but changing the behavior of the test would probably make it contradictory to it.

Reply:
no.  def don't delete the workitem.  add your *new* workitem to it.  this allows us to identify which scenarios led to the behavior and to track the items tha tcaused it to change.  note: if you do `>=2` i would not expect this scenario to change.  I would also expect your `"i want to match 'id' "` case to now work.